### ISSUE TITLE
Interactive web graphics

### ISSUE THREAD

**Issue by durack1** [PCMDI/pcmdi_metrics#357]
I just noted the nifty javascript interactive web graphics at:

http://www.abc.net.au/news/2016-05-03/budget-2016-sliced-diced-interactive/7363834#spending/breakdown/2017/general-public-services

@chaosphere2112 do you have much experience with interactive graphics? I do think such snazzy stuff would be super useful in the ongoing metrics development..

**Comment by chaosphere2112** [comment]
@durack1 Yeah, I've done interactive visualization for the browser (I did [this](http://www.sambfries.me/HopViz/) a little bit ago). You can get a lot of whizz-bang for your buck, and it's not very hard to assemble. They're better for exploring data, trying to find interesting things that you don't know are there (versus traditional static visualization). Having a tool for interactively exploring climate data might be nice for discovering trends/relationships that would make for good metrics, but probably would be iffy as an actual "metric" itself (where theoretically you're looking for glance-able data). Also good for advertising (whether to the public, to funding agencies, or to users).  That's my 2¢, at least.

**Comment by durack1** [comment]
@chaosphere2112 exactly, the whizz-bang aspect is useful.. It's also a nice way to capture a large amount of data in a fairly compact format.

Good to know you've had some experience with this..

**Comment by doutriaux1** [comment]
@jservonnat isn't it what you and Patrick Brockmann are trying to do?

**Comment by durack1** [comment]
I just spotted this graphic and thought it was an interesting way of capturing info
![reproducibility-graphic-online2](https://cloud.githubusercontent.com/assets/3229632/15691086/b59619c8-273b-11e6-8146-cb3a7d883a00.jpg)

**Comment by doutriaux1** [comment]
@gleckler1 it's an enhancement for 2.0. Why close this?

**Comment by gleckler1** [comment]
@doutriaux1 ok to stay open but its clutter among things we need to get done on short term.

**Comment by durack1** [comment]
@gleckler1 unclutter your issue list using the milestone tag:
https://github.com/PCMDI/pcmdi_metrics/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.1
![1p1](https://cloud.githubusercontent.com/assets/3229632/16398691/3104e382-3c81-11e6-837e-b5fa29b23cd8.png)

If we don't have multiple milestones there'll be no other way to capture information and separate this against targeted releases

**Comment by doutriaux1** [comment]
not really, just sort by milestone.
